javascript/routes/index.js

```diff
@@ -57,6 +62,51 @@ router.get('/redirect', (req, res) => {
   res.redirect(uri)
 })
 
+// example implementation of creating a PASETO customer token
+router.get("/redirect/paseto", (req, res) => {
+  let token;
+  axios
+    .post(
+      CK_CUSTOMER_TOKEN_ENDPOINT,
+      {},
+      {
+        auth: {
+          username: CLIENT_ID,
+          password: CLIENT_SECRET,
+        },
+        params: {
+          email: "sam.pull@example.com",
+          first_name: 'Samuel',
+          last_name: 'Pull',
+          custom_reference: 'SP-123',
+          date_of_birth: '1985-10-25',
+          postcode: 'XY12AB',
+        },
+      }
+    )
+    .then(function (response) {
+      token = response.data.data.customerToken;
+      console.log("response: ", response);
```

Comment:
Are the console logs on here, L92 and L105 intentional or left over from testing? Unsure here given the purpose of the repo

Reply:
it's following the same pattern used for the JWT customer token implementation just above this route. I think both would be useful for a client to see when testing the implementation